spec/descriptions/tagMaintenanceConfiguration.md

```diff
@@ -0,0 +1,23 @@
+### Maintenance Windows
+Maintenance windows allow you to mute alerts.
+
+### Pausing Maintenance Windows
+Maintenance windows can be paused and resumed.
+
+If a maintenance window is paused, alerts will no longer be muted. 
+
+_NOTE: Pausing unmutes alerts. Maintenance Status is separate from pausing a maintenance window._
+
+### Types of Maintenance Configurations
```

Comment:
Can you move this section before the Pause/Resume one?
Also, can you provide a short description for each? 
One-Time: executed once at the fixed scheduled time, after which they expire
Recurrent: executed on a schedule basis for planned maintenance activities